src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/reductions/GetReductionServiceTests.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.integration.reductions
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.repository.LduRepository
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.repository.RegionRepository
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.LduEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.OffenderManagerEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.ReductionEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.ReductionStatus
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.RegionEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.TeamEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.WMTWorkloadOwnerEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.reduction.GetReductionService
+import java.math.BigDecimal
+import java.time.ZonedDateTime
+
+/*
+   temporary test until we have an orchestrator. For orchestrator, I thought:
+   1. get all out of date reductions
+   2. Update all out of date reductions
+   3. Map them to a distinct list of staff code, team code, loop through, run calculation
+   4. emit event for reduction status finished to kick off wmt-worker ETL process
+ */
+class GetReductionServiceTests : IntegrationTestBase() {
+
+  @Autowired
+  protected lateinit var regionRepository: RegionRepository
+
+  @Autowired
+  protected lateinit var lduRepository: LduRepository
+
+  @Autowired
+  protected lateinit var getReductionService: GetReductionService
+
+  @Test
+  fun `must return all active reductions which have an effective to in the past`() {
+    val region = regionRepository.save(RegionEntity(code = "REGION1", description = "Region 1"))
+    val pdu = lduRepository.save(LduEntity(code = "LDU1", description = "Local Delivery Unit (Actually a Probation Delivery Unit)", region = region))
+    val team = teamRepository.save(TeamEntity(code = "TEAM1", description = "Team 1", ldu = pdu))
+    val offenderManager = offenderManagerRepository.save(OffenderManagerEntity(code = "STAFFCODE1", forename = "Jane", surname = "Doe", typeId = 1))
+    val workloadOwner = wmtWorkloadOwnerRepository.save(WMTWorkloadOwnerEntity(offenderManager = offenderManager, team = team, contractedHours = BigDecimal.valueOf(37.5)))
+
+    val activeReductionInPast = reductionsRepository.save(ReductionEntity(workloadOwner = workloadOwner, hours = BigDecimal.valueOf(3.2), effectiveFrom = ZonedDateTime.now().minusDays(7), effectiveTo = ZonedDateTime.now().minusDays(1), status = ReductionStatus.ACTIVE, reductionReasonId = 1))
```

Comment:
might be an idea to stick a current reduction in here to make sure we have the JPA right?

Reply:
I've also added a deleted in the past too